src/Akeneo/Bundle/ElasticsearchBundle/Cursor/AbstractCursor.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Akeneo\Bundle\ElasticsearchBundle\Cursor;
+
+use Akeneo\Bundle\ElasticsearchBundle\Client;
+use Akeneo\Component\StorageUtils\Cursor\CursorInterface;
+use Akeneo\Component\StorageUtils\Repository\CursorableRepositoryInterface;
+
+/**
+ * Common logic shared by all our cursors.
```

Comment:
it's in the Elasticsearch namespace, of course it's for the Elasticsearch cursors